command/loadbot/result.go

```diff
@@ -42,11 +48,14 @@ type TxnDetailedErrorData struct {
 }
 
 type LoadbotResult struct {
-	CountData         TxnCountData         `json:"count_data"`
-	TurnAroundData    TxnTurnAroundData    `json:"turn_around_data"`
-	BlockData         TxnBlockData         `json:"block_data"`
-	DetailedErrorData TxnDetailedErrorData `json:"detailed_error_data,omitempty"`
-	ApproxTPS         uint64               `json:"approx_tps"`
+	CountData              TxnCountData         `json:"count_data"`
+	TurnAroundData         TxnTurnAroundData    `json:"turn_around_data"`
+	ContractTurnAroundData TxnTurnAroundData    `json:"contract_turn_around_data"`
+	BlockData              TxnBlockData         `json:"block_data"`
+	DetailedErrorData      TxnDetailedErrorData `json:"detailed_error_data,omitempty"`
+	ApproxTPS              uint64               `json:"approx_tps"`
+	ContractAddress        web3.Address         `json:"contract_address"`
+	ContractBlockData      TxnBlockData         `json:"contract_block_data"`
```

Comment:
What do you think about making these omittable in case the mode is transfer?

cc @Kourin1996

Reply:
Fixed in 912c140f691ad1c9fd3a01e65b4de25086526eb2